app/webpacker/controllers/enterprise_form_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = [ "form", "default" ]
+
+  connect() {
+    // hide all forms by default
+    this.formTargets.forEach((form) => this.hide(form) )
+    this.showDefaultForm()
+  }
+
+  showDefaultForm() {
+    this.defaultTarget.style.display = "block"
+  }
+
+  hide(form) {
+    form.style.display = "none"
+  }
+
+  show({detail: { currentActiveTab, newActiveTab }}) {
+    let currentActiveForm = this.element.querySelector(`#${currentActiveTab.id}_form`)
+    let newActiveForm = this.element.querySelector(`#${newActiveTab.id}_form`)
```

Comment:
I'd vote to use a `target`. here, but I'm honest, not sure to understand how it works, and how is the event built.